src/js/App/RootApp.js

```diff
@@ -3,73 +3,58 @@ import PropTypes from 'prop-types';
-}) => {
-    const isGlobalFilterEnabled = !globalFilterHidden && (
-        window?.insights?.chrome?.isBeta() || Boolean(localStorage.getItem('chrome:experimental:global-filter'))
-    ) &&
+const RootApp = ({ activeApp, activeLocation, appId, pageAction, pageObjectId, globalFilterHidden }) => {
+  const isGlobalFilterEnabled =
+    !globalFilterHidden &&
+    (window?.insights?.chrome?.isBeta() || Boolean(localStorage.getItem('chrome:experimental:global-filter'))) &&
     location.pathname.includes('insights');
-    return (
-        <Fragment>
-            <div
-                className="pf-c-drawer__content"
-                data-ouia-page-type={activeApp || appId}
-                data-ouia-page-group={activeLocation}
-                data-ouia-safe='true'
-                {...pageAction && { 'data-ouia-page-action': pageAction }}
-                {...pageObjectId && { 'data-ouia-page-object-id': pageObjectId }}
-            >
-                <div>
-                    {isGlobalFilterEnabled && <GlobalFilter />}
-                    <main className="pf-c-page__main pf-l-page__main" id="root" role="main">
-                        <section
-                            className="pf-m-light pf-c-page-header pf-c-page__main-section pf-m-light"
-                            widget-type="InsightsPageHeader"
-                        >
-                            <div className="pf-c-content">
-                                <h1
-                                    className="pf-c-title pf-m-2xl ins-l-page__header--loading"
-                                    widget-type='InsightsPageHeaderTitle'
-                                >
-                                    <div className="ins-c-skeleton ins-c-skeleton__sm">
-                                    &nbsp;
-                                    </div>
-                                </h1>
-                            </div>
-                        </section>
-                        <section className="pf-c-page__main-section pf-l-page__main-section--loading">
-                            <div className="ins-c-spinner ins-m-center" role="status">
-                                <span className="pf-u-screen-reader">Loading...</span>
-                            </div>
-                        </section>
-                    </main>
-                    <main className="pf-c-page__main" id="no-access"></main>
-                </div>
-            </div>
-            <aside className="pf-c-drawer__panel">
-                <div className="pf-c-drawer__panel-body" />
-            </aside>
-        </Fragment>
-    );
+  return (
+    <Fragment>
+      <div
+        className="pf-c-drawer__content"
+        data-ouia-page-type={activeApp || appId}
+        data-ouia-page-group={activeLocation}
+        data-ouia-safe="true"
+        {...(pageAction && { 'data-ouia-page-action': pageAction })}
+        {...(pageObjectId && { 'data-ouia-page-object-id': pageObjectId })}
+      >
+        <div>
```

Comment:
Addition of this div breaks the PF Page + PageSection layout which uses flexbox. This currently breaks the Assisted Installer in OCM. It needs to be replaced with React.Fragment

Reply:
This PR uses new linter, you might be refering to this PR https://github.com/RedHatInsights/insights-chrome/pull/1008